src/pulumi/pulumi_examples.md

```diff
@@ -0,0 +1,107 @@
+<h1> Deployment Examples</h1>
+
+![](./img/pulumi_logo.svg)
+
+<h2>Table of Contents</h2>
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Set the Environment Variables](#set-the-environment-variables)
+- [Test the Plugin](#test-the-plugin)
+- [Destroy the Deployment](#destroy-the-deployment)
+- [Questions and Feedback](#questions-and-feedback)
+
+***
+
+## Introduction
+
+[Pulumi](https://www.pulumi.com/) is an infrastructure as code platform that allows you to use familiar programming languages and tools to build, deploy, and manage cloud infrastructure.
+
+We present here the basic steps to test the examples within the [ThreeFold Pulumi](https://github.com/threefoldtech/pulumi-threefold) plugin repository. Once you've set the plugin and exported the necessary variables, the deployment process from one example to another is very similar.
+
+Please note that the Pulumi plugin for ThreeFold Grid is not yet officially published. We look forward to your feedback on this project.
+
+## Prerequisites
+
+There are a few things to set up before exploring Pulumi.
+
+* Install [Go](https://go.dev/)
```

Comment:
You should be able to use it directly without building the plugin manually

Reply:
Let's simplify the installation and we can merge

Reply:
Good. Let me know if this is ok. Make the step "make build" isn't necessary. Please check and I can adjust if needed.

Reply:
@xmonader I think we don't need make build, since as you say we don't need to build the plugin manually.

So I just removed the line make build. Let me know if there's anything more that needs to be done. thanks!